design/sketch.webidl

```diff
@@ -137,13 +133,12 @@ interface WebGPUShaderStageBit {
     const u32 COMPUTE = 4;
 };
 
-typedef u32 WebGPUBindingTypeEnum;
-interface WebGPUBindingType {
-    const u32 UNIFORM_BUFFER = 0;
-    const u32 SAMPLER = 1;
-    const u32 SAMPLED_TEXTURE = 2;
-    const u32 STORAGE_BUFFER = 3;
-    // TODO other binding types (storage images, ...)
+enum WebGPUBindingType {
+    "uniformBuffer",
```

Comment:
> It is strongly suggested that enumeration values be all lowercase, and that multiple words be separated using dashes or not be separated at all, unless there is a specific reason to use another value naming scheme. For example, an enumeration value that indicates an object should be created could be named "createobject" or 'create-object". Consider related uses of enumeration values when deciding whether to dash-separate or not separate enumeration value words so that similar APIs are consistent.